src/App.js

```diff
@@ -1,12 +1,36 @@
-import './css/App.css';
+import { useState, useEffect } from 'react';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import './css/style.css';
+import axios from 'axios';
```

Comment:
You need to add axios to your package.json

Reply:
added in 4ac62df74c5bdab5927a623d702adcd0e14dafe6